docfiles/footer.html

```diff
@@ -20,12 +20,12 @@
         </div>
         <div class="ui section divider"></div>
         <div class="ui container horizontal small divided link list">
+            <a id="langpicker" tabindex="0" class="ui item" rel="noopener">
+                <i class="icon xicon globe" aria-hidden="true" role="presentation"></i>Language
+            </a>
             <a class="item" href="https://makecode.com/privacy" target="_blank" rel="noopener">Privacy &amp; Cookies</a>
             <a class="item" href="https://makecode.com/termsofuse" target="_blank" rel="noopener">Terms Of Use</a>
             <a class="item" href="https://makecode.com/trademarks" target="_blank" rel="noopener">Trademarks</a>
-            <!-- @ifdef crowdinproject -->
```

Comment:
why is this removed?

Reply:
I bumped it to the bottom of the lang selection page to match editor intro, was going to ask you about that on monday though.  Will put it back here if you prefer